File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/NetworkDeleteByNameTest.java
package com.sequenceiq.it.cloudbreak;

import org.testng.annotations.Parameters;
import org.testng.annotations.Test;

public class NetworkDeleteByNameTest extends AbstractCloudbreakIntegrationTest {
    @Test
    @Parameters({ "networkName" })
    public void testDeleteTemplateByName(String networkName) throws Exception {
        // GIVEN
        // WHEN
        getCloudbreakClient().networkEndpoint().deletePublic(networkName);
        // THEN no exception
    }
}

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/NetworkDeleteByNameTest.kt
package com.sequenceiq.it.cloudbreak

import org.testng.annotations.Parameters
import org.testng.annotations.Test

class NetworkDeleteByNameTest : AbstractCloudbreakIntegrationTest() {
    @Test
    @Parameters("networkName")
    @Throws(Exception::class)
    fun testDeleteTemplateByName(networkName: String) {
        // GIVEN
        // WHEN
        cloudbreakClient.networkEndpoint().deletePublic(networkName)
        // THEN no exception
    }
}
